src/assets/i18n/en.json5

```diff
@@ -1676,6 +1684,28 @@
 
 
 
+  "item.listelement.badge": "Item",
```

Comment:
Similarly, I don't think we need a badge for every default Item.  I'd only expect badges to appear for typed items (similar to how they already do in current `main` code and https://dspace7-demo.atmire.com/).  To me, badges should only appear to call out differences, and not automatically appear on everything in a list...this appears to change that behavior.

As a sidenote, there appears to be a bug around badges anyways.  Currently, when I run this PR, I see a badge that says `t.listelement.badge` on every untyped Item.

Reply:
>Similarly, I don't think we need a badge for every default Item. I'd only expect badges to appear for typed items (similar to how they already do in current main code and https://dspace7-demo.atmire.com/). To me, badges should only appear to call out differences, and not automatically appear on everything in a list...this appears to change that behavior.

I asked yana to add those same badges to all list elements because it struck me as confusing for an end user that some results would have them and others in the same list wouldn't. I know I find it hard myself to determine if a search result is a community, collection or item without those badges.

>Currently, when I run this PR, I see a badge that says t.listelement.badge on every untyped Item.

That's likely a consequence of minification. The class 'Item' will likely be minified to 'T' in prod mode. We can fix that by using the type property instead of the class name

Reply:
@artlowel : That's reasonable. We can keep the badges on every object for now and see how end users respond.  So, we just need to fix the bug which displays `t.listelement.badge` instead of `Item` for untyped items.